### ISSUE TITLE
Allow arbitrary default-encoding via commandline param 

### ISSUE THREAD

**Issue by frisi** [zopefoundation/zodbupdate#9]
plonesites should use utf-8 at most places, but here in halle we heard about other companies that are using latin-1 as default encoding.
to support migrating these datbases i suggest to add a command line option used combined with `--convert-py` that allows to tell zodbupdate which encoding to use when converting python2 str to python3 str:

```zodbupdate --convert-py --default-encoding='latin-1'```

actually it would also be better to rename the marker for python 3 `str` (formerly python2 unicode) from `utf-8`  to something that else as it might also stand for `latin-1` or other encodings:

```
#old:
decode_dict = {
   'mypackage.mymodule ClassName attribute': 'binary',
   'otherpackage.othermodule OtherClass other_attribute': 'utf-8'}

#new:
decode_dict = {
   'mypackage.mymodule ClassName attribute': 'binary',
   'otherpackage.othermodule OtherClass other_attribute': 'encoded_str'}
```

**Comment by thefunny42** [comment]
I have the feeling you are confusing "unicode" and "utf-8". They is has nothing to do with each other. The "utf-8" option here is to decoded an encoded Python2 ``str``.  I believe you want to keep this. I would only add a "default" that is basically the value you pass to the script. After I am not sure this strategy will fly: if people uses "latin-1" they actually probably have a mix of "utf-8" and "latin-1" in their databases, and a command line option is not going to cut it.

**Comment by icemac** [comment]
@frisi I think this has been fixed in #14, is this correct?

**Comment by frisi** [comment]
reading the conversation in #14 it seems this pretty much does what was outlined here. not sure if one can define attributes should be converted to strings and which shoud stay bytes but maybe the automatic guessing works and saves us from writing definitions for all string attributes.
i can't test this currently but i'm confident that if it worked for the sprinters in march (as philipp reported) it will do the trick and this issue can be considered solved.
